lib/unit-ruby/util/api_resource.rb

```diff
@@ -83,8 +83,9 @@ def resource_type
 
     # Creates an association to a related resource
     # This will create a helper method to traverse into a resource's related resource(s)
-    def self.belongs_to(resource_name, class_name: nil)
+    def self.belongs_to(resource_name, resource_type: nil, class_name: nil)
       class_name ||= resource_name.to_s.camelize
+      resource_type ||= resource_name.to_s.camelize(:lower)
```

Comment:
this is kinda just future proofing -- all `belongs_to` associations right now are either a single lowercase resource_name, or have an explicit resource_type defined, but in theory this should let you do something like `belongs_to :application_form` and `resource_type` will be `applicationForm` without being explicitly defined